aait/web/group-4/starter-project/app/layout.tsx

```diff
@@ -1,8 +1,10 @@
+import Navbar from "@/components/common/Navbar";
 import "./globals.css";
```

Comment:
You don't have to import global.css file, it is better to remove it.

Reply:
tailwindcss won't work if i do that